detail.py

```diff
@@ -0,0 +1,38 @@
+from django.conf import settings
+from django_mako_plus import view_function, jscontext
+from catalog import models as cmod
+import math
+
+@view_function
+def fix_html(x):
+    x = x.replace("&","&amp;")
+    x = x.replace("<","&lt;")
+    x = x.replace(">","&gt;")
+    return x
+
+@view_function
+def process_request(request, product: cmod.Product=None):
+
+    p_id = product.id
+    p_name = fix_html(product.name)
```

Comment:
Neither of these variables are used in the function. It appears they are unnecessary.

Reply:
ohhh I might need to add the p_name back in context to call this method on the variable before sending it over